src/main/java/frc/robot/constants/RobotConfig.java

```diff
@@ -3,14 +3,130 @@
+
+    // Smart dashboard Angle Controller keys
+    public static final String kAngleControlPGainKey = "Angle Controller P Gain";
+    public static final String kAngleControlIGainKey = "Angle Controller I Gain";
+    public static final String kAngleControlDGainKey = "Angle Controller D Gain";
+    public static final String kAngleControlFFGainKey = "Angle Controller FF Gain";
+    public static final String kAngleControlIZoneKey = "Angle Controller I Zone";
+    public static final String kAngleControlMinOutputKey = "Angle Controller Minimum output";
+    public static final String kAngleControlMaxOutputKey = "Angle Controller Maximum output";
+
+    // Smart dashboard top Flywheel Controller keys
+    public static final String kTopFlywheelPGainKey = "Top Flywheel P Gain";
+    public static final String kTopFlywheelIGainKey = "Top Flywheel I Gain";
+    public static final String kTopFlywheelDGainKey = "Top Flywheel D Gain";
+    public static final String kTopFlywheelFFGainKey = "Top Flywheel FF Gain";
+    public static final String kTopFlywheelIZoneKey = "Top Flywheel I Zone";
+    public static final String kTopFlywheelMinOutputKey = "Top Flywheel Minimum output";
+    public static final String kTopFlywheelMaxOutputKey = "Top Flywheel Maximum output";
+
+    // Roller Default Speed
+    public static final double kRollerDefaultSpeed = -0.4;
+    // Flywheel default speed
+    public static final double kFlywheelDefaultRPM = 1000;
+    // Shield Retracted position
+    public static final double kShieldRetractedPosition = 15;
+    // Timeout time (in seconds)
+    public static final double kRunIntakeTimeoutTime = 0;
+    public static final double kShieldExtendedPosition = 85; // TODO get correct value
+    // Speaker height
+    public static final double SpeakerHeight = 1.9812;
+    public static final double AmpHeight = .46;
+    public static final double ShooterHeight = 0.28575;
+    public static final double TrapHeight = 1;
+
+    public static final double SpeakerBillLength = 0.6604;
+
+    public static final int shootButton = 1;
+
+    public static final double kDefaultFlywheelRPM = 1000;
+
+    public static final double kShooterStowAngle = 0;
+
+    public static final long kReleaseTime = 5000;
+    public static final long kShieldTime = 2; // seconds
+    public static final double kShieldDefaultSpeed = 0.5;
+    public static final Measure<Velocity<Angle>> kFlywheelError = Units.RPM.of(1);
+    public static final Measure<Angle> kAngleError = Units.Radians.of(0.5 * Math.PI / 180);
+    public static final Measure<Angle> kSpeakerAngle = Units.Radians.of(75 * Math.PI / 180);
+    public static final Measure<Angle> kAmpAngle = Units.Radians.of(109 * Math.PI / 180);
+    public static final Measure<Angle> kTrapAngle = Units.Radians.of(105 * Math.PI / 180);
+    public static final Measure<Angle> kAdjustAmountDegrees = Units.Radians.of(0.5 * Math.PI / 180);
+
+    // TODO placeholders
```

Comment:
why are these so low and are they still placeholders and are they being used in the flywheel speed

Reply:
These are the correct values. Although they are pretty weird.